cc/selfplay.cc

```diff
@@ -94,6 +94,16 @@ DEFINE_string(flags_path, "",
               "Note that flags_path is different from gflags flagfile, which "
               "is only parsed once on startup.");
 
+DEFINE_double(fastplay_frequency, 0.0,
+              "The fraction of moves that should use a lower number of "
+              "playouts, aka 'playout cap oscillation'.\nIf this is set, "
+              "'fastplay_readouts' should also be set.");
+
+DEFINE_int32(fastplay_readouts, 20,
```

Comment:
is there a pattern that avoids potentially having conflicting defaults in the flag parser and the options constructor?

Reply:
Not that i know of.